garak/probes/glitch.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 """Glitch keyword probes
 
-Testing for tokens that lead to unusual model behaviour
```

Comment:
UK/US spelling again.

Reply:
Just for consistency as both spellings are in this file.